tests/test_gallery.py

```diff
@@ -114,6 +114,7 @@ def test_image(settings, tmpdir):
 
 def test_video(settings, tmpdir):
     settings['destination'] = str(tmpdir)
+    settings['use_orig'] = False
```

Comment:
This surprised me a bit, `use_orig` should be False by default, but it seems that the settings dict is modified by `test_media_orig`, which is bad. I will try to fix this (unless you want to take a look :-)).